app/validation.py

```diff
@@ -226,3 +222,20 @@ def is_valid_date(date, default_format=DATE_FORMAT):
 
 def is_valid_acknowledged_state(acknowledged):
     return acknowledged in ['all', 'true', 'false']
+
+
+def is_valid_string_or_400(string):
+    if not is_valid_string(string):
+        abort(400, "invalid value {}".format(string))
+
+
+def is_valid_string(string, minlength=1, maxlength=255):
+    regex_match_valid_service_id = r"^[A-z0-9-]{%s,%s}$" % (
```

Comment:
Should the name of this regex change if we're using it to check more than just service IDs now?